src/components/map/map.js

```diff
@@ -640,10 +643,12 @@ class Map extends React.Component {
     const maxZoom = this.getMaxZoomForFittingMapToData();
     // first, clear any existing timeout
     if (this.bounds_timeout) {
-      window.clearTimeout(this.bounds_timeout);
+      removeTimeout(this.bounds_timeout);
```

Comment:
needs to be `removeTimeout('map', this.map_timeout)`

Reply:
oops, sorry